### DISCUSSION TITLE
Customized SharePoint PowerApp Form

### DISCUSSION THREAD

**Discussion by vandeso** [Zerg00s/FlowPowerAppsMigrator#11] [General]
Hi,

Great tool! I had to migrate a customized SharePoint list form in PowerApps from a site collection to another within the same tenant. I found this tool and thought I give it a try while it's actually developed for canvas app.
Guess what it works perfectly for customized SharePoint list for forms in PowerApps as well!
Something to add on your features list.

Thank you for creating this great tool! A time-saving app!

**Comment by Zerg00s** [comment]
Hi @vandeso! Thank you very much. I am really happy the Flow & App Migrator works. I used this tool for migrating customized list forms too, but I forgot to mention it anywhere in the readme file. Now I made sure this information is included. Migrating customized forms must be one the most popular use cases.

**Comment by bucifan** [comment]
Hi, We are hoping to use this tool to migrate customized SharePoint PowerApps forms.  In my testing I have found that this process will only work if the source and destination SharePoint list have the same internal name.  So if I wanted to move a SharePoint PowerApps form from Site 1 - form assigned to _list A_ to a destination like Site 2 _some other named list B_.  This tool cannot yet support this - is this correct?

**Reply by Zerg00s** [reply]
Hi @bucifan . Correct. The tool needs the internal names to match.
